src/OpenLoco/src/Paint/PaintStation.cpp

```diff
@@ -200,18 +200,20 @@ namespace OpenLoco::Paint
         // Paint Back part of platform
         {
             World::Pos3 bbOffset = platformImages.back.bbOffset + heightOffset;
-            session.addToPlotList4FD150(imageBase.withIndexOffset(platformImages.back.imageId), heightOffset, bbOffset, platformImages.back.bbSize);
 
             paintStationCargo(session, elStation, cargoFlags[0], 0xFFFFFFFF, cargoOffsets, elStation.baseHeight(), bbOffset, platformImages.back.bbSize);
             paintStationCargo(session, elStation, cargoFlags[1], 0xFFFFFFFF, cargoOffsets, elStation.baseHeight(), bbOffset, platformImages.back.bbSize);
+
+            session.addToPlotList4FD150(imageBase.withIndexOffset(platformImages.back.imageId), heightOffset, bbOffset, platformImages.back.bbSize);
```

Comment:
this is a bit odd. Before it was:
parent = platform back
children = cargo1 back
children children = cargo 2 back
parent = platform front
children = cargo1 front
children children = cargo 2 front

Now it will be:
parent = ???
children = cargo1 back (will be premoted to parent if no previous parent available)
children children = cargo 2 back
parent = platform back
children = cargo1 front
children children = cargo 2 front
parent = platform front

So you will have issues if anything gets placed in the middle (like the train).
I think what you want to do is make the first cargo a parent in all cases so its:
parent = cargo1 back (will be premoted to parent if no previous parent available)
children = cargo 2 back
parent = platform back
parent = cargo1 front
children = cargo 2 front
parent = platform front